molecule/verifier/lint/precommit.py

```diff
@@ -0,0 +1,143 @@
+#  Copyright (c) 2015-2018 Cisco Systems, Inc.
```

Comment:
Think this needs to be `Copyright (c) 2018‒2019 Red Hat, Inc.`. See other more recently edited files.

Reply:
A lot of this code was directly copied from the `flake8.py` in the same directory and modified from there. What suggestions should I apply to `flake8.py` as well?

Reply:
flake8 was developed before the acquisition of Molecule by Ansible Project/Red Hat. So it only applies to new code. But It shouldn't be `2015-`, because you didn't write it 4 years ago.

Reply:
Okay, thanks for the fix I'm going to call this one resolved.